tests/test_rrd.py

```diff
@@ -33,10 +33,10 @@ def test_2_points_out_of_3(self, get_rrd_method):
 
         def update_rrd(multiplier):
             """Update rrd."""
-            rx = tx = multiplier * STATS_INTERVAL
-            tstamp = start + rx
+            rx_arg = tx_arg = multiplier * STATS_INTERVAL
+            tstamp = start + rx_arg
             # any value for index is OK
-            rrd.update([None], tstamp, rx=rx, tx=tx)
+            rrd.update([None], tstamp, rx=rx_arg, tx=tx_arg)
```

Comment:
RX and TX are not really arguments to this method. Maybe `rx_val` or `rx_value` are better names.

Reply:
I updated the files.